src/libcore/slice.rs

```diff
@@ -655,6 +655,12 @@ impl<'a, T> Default for &'a [T] {
     fn default() -> &'a [T] { &[] }
 }
 
+#[stable(feature = "mut_slice_default", since = "1.5.0")]
+impl<'a, T> Default for &'a mut [T] {
+    #[stable(feature = "mut_slice_default", since = "1.5.0")]
```

Comment:
Consistency until purged. Also doesn't hurt and is better if down the track the annotations are respected as intended.